chart/kubeapps/templates/dashboard-config.yaml

```diff
@@ -17,6 +17,15 @@ data:
       gzip_static  on;
 
       location / {
+        # Support for ingress prefixes maintaining compatibility with the default /
+        # 1 - Exactly two fragment URLs for files existing inside of the public/ dir
+        # i.e /[prefix]/config.json => /config.json
+        rewrite ^/[^/]+/([^/]+)$ /$1 break;
+
+        # 2 - Any static files bundled by webpack
+        # i.e /[prefix]/static/main.js => static.main.js
```

Comment:
either the comment is wrong or I am not understanding the regex, wouldn't this be:
```
         # i.e /[prefix]/static/main.js => /static/main.js
```
Also, isn't this regex already caught in the first?

Reply:
This comment is wrong, thanks for catching it up!